src/validation.ts

```diff
@@ -144,14 +144,35 @@ export const validateTLSServerName = (serverName: string) => {
 };
 
 export const validateTLSCACert = (caCert: string) => {
+  if (!caCert) {
+    return undefined;
+  }
+  if (caCert.indexOf('-----BEGIN CERTIFICATE-----') < 0 || caCert.indexOf('-----END CERTIFICATE-----') < 0) {
```

Comment:
If you use optional chaining here, you can get rid of the conditional block above.